.github/workflows/wheels-dependencies.sh

```diff
@@ -134,7 +134,7 @@ function build_libavif {
 
     local out_dir=$(fetch_unpack https://github.com/AOMediaCodec/libavif/archive/refs/tags/v$LIBAVIF_VERSION.tar.gz libavif-$LIBAVIF_VERSION.tar.gz)
     # CONFIG_AV1_DECODER=0 is a flag for libaom (included as a subproject of
-    # libavif) to disable the compilation and inclusion of aom's AVIF decoder.
+    # libavif) to disable the compilation and inclusion of aom's AV1 decoder.
```

Comment:
I also suggest moving `-DENABLE_NASM=ON` next to `-DCONFIG_AV1_DECODER=0` because it is also a cmake flag for libaom.

Why do you need to set `ENABLE_NASM` to `ON`? Are you concerned that yasm may be used?

Reply:
I think it's probably unnecessary. That flag goes back to when I first opened #5201, over four years ago. If there was a reason for its inclusion then (and there may not have been) I have forgotten it.

Reply:
`yasm` won't be in the `PATH` in any of the build environments, so even if it was a matter of forcing the user of nasm over yasm, it wouldn't be necessary. I'll remove it.